Vagrantfile

```diff
@@ -9,7 +9,7 @@ Vagrant.configure("2") do |config|
   # The most common configuration options are documented and commented below.
   # For a complete reference, please see the online documentation at
   # https://docs.vagrantup.com.
-
+  
```

Comment:
this looks like an unnecessary edit

Reply:
Did we require to add these 3 lines?

 config.vm.network(
  "forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
  )
  
 What should be change?

Reply:
@shayakrach, I think what Yuval is saying is that line 11 and 12 are not relevant they are just blank white lines.